### DISCUSSION TITLE
Problems with activating the "GitHub Education benefits"

### DISCUSSION THREAD

**Discussion by ghmbacher** [community/community#142380] [GitHub Education]
### Hi everyone,

On Oct 21, 2024 I submitted 2 times. 

My first application was declined because I didn't fill out my billing information and didn't have 2-factor authentication.
I added the missing things to my account and submitted another application, this one was accepted. 
Since then I have not received any more information. I know the website says it takes up to 3 days, sometimes longer, but that's not normal, is it? Shouldn't I at least have received a confirmation email?

Thanks for any answers in advance!
<img width="323" alt="Bildschirmfoto 2024-10-23 um 12 19 08" src="https://github.com/user-attachments/assets/c37c0fd8-6498-46cd-b446-e3d7c079eaa8">

**Comment by lin-ycv** [comment]
it can take way longer than 3 days.
similar topic here:
https://github.com/orgs/community/discussions/142366
